File: Beecrowd/Beginner/1101 - 1190/1188 - Inferior Area/C++/1188_InferiorArea.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    char operation;
    double matrix[12][12], sum = 0;
    cin >> operation;

    for (int i = 0; i < 12; i++) {
        for (int j = 0; j < 12; j ++) {
            cin >> matrix[i][j];

            if (i >= 7 && j >= 12 - i && j <= i - 1) {
                sum += matrix[i][j];
            }
        }
    }

    if (operation == 'S') cout << fixed << setprecision(1) << sum << endl;
    else cout << fixed << setprecision(1) << sum / 30 << endl;

    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1188 - Inferior Area/Python/1188_InferiorArea.py
sum = 0

operation = input()

for i in range(12):
    for j in range(12):
        number = float(input())
        if i >= 7 and j >= 12 - i and j <= i - 1:
            sum += number
            
if operation == 'S':
    print(f"{sum:.1f}")
else:
    print(f"{sum / 30:.1f}")
